src/status_im/contexts/keycard/different_card/view.cljs

```diff
@@ -14,7 +15,9 @@
     {:title            (i18n/label :t/different-keycard)
      :description      :text
      :description-text (i18n/label :t/scan-previous-keycard)}]
```

Comment:
Only the illustrations have been updated; we are still using the same description text in the scope of the PR. However, if we check the linked issues in the PR, it seems that different keycards are using different description texts based on the context.